lib/zclient.c

```diff
@@ -287,7 +288,7 @@ static int zclient_flush_data(struct thread *thread)
 			(*zclient->zebra_buffer_write_ready)();
 		break;
 	}
-	return 0;
+	return;
```

Comment:
it looks for few void  apis return statement added at the end of the functions, is there any specific reason ?
I feel ,it would be good  if it is consistent.

Reply:
fixed